Observer/BeforeSending.php

```diff
@@ -34,6 +34,8 @@ public function execute(Observer $observer): void
         if ($hint?->exception instanceof LocalizedException) {
             $hintMessage = $hint->exception->getRawMessage();
             $event->setMessage($hintMessage);
+            $event->getExceptions()[0]->setValue($hintMessage);
```

Comment:
It may be a good idea to check if this index exists

Reply:
@indykoning There should be at least one exception in the array. Sentry prepares the event and does the following checks before setting the exceptions on the event: https://github.com/getsentry/sentry-php/blob/master/src/Client.php#L244 and https://github.com/getsentry/sentry-php/blob/master/src/Client.php#L415C22-L415C41

I can add the check for the index to be sure.